src/constants/api.ts

```diff
@@ -0,0 +1,9 @@
+export const BASE_URL = "https://panda-market-api.vercel.app";
+
+export const PATH = {
+  ME: "users/me",
+  PRODUCTS: "products",
+  SIGNUP: "auth/signUp",
+  LOGIN: "auth/signIn",
+  REFRESH: "auth/refresh-token",
+} as const;
```

Comment:
P3:
두개를 사용처에서 조합해서 사용하는 것 보다 아래처럼 하시는게 더 좋지 않을까싶네요.
분리해야할 이유를 모르겠어서요~

```suggestion
export const PATH = {
  ME: `${BASE_URL}/users/me`,
  PRODUCTS: `${BASE_URL}/products`,
  SIGNUP: `${BASE_URL}/auth/signUp`,
  LOGIN: `${BASE_URL}/auth/signIn`,
  REFRESH: `${BASE_URL}/auth/refresh-token`,
} as const;
```